<!-- It will verify the gnss covariances sensor_msgs/NavSatFix and republish the 
same gnss data but modifing the status to 2 (ground-based augmentation)  -->
<launch>

  <arg name="gnss_topic_in" default="/uav1/gnss_verifier/gnss_wstatus"/>
  <arg name="base_station_latitude" default="47.397743"/>
  <arg name="base_station_longitude" default="8.545594"/>
  <arg name="frame_id" default="world1"/>
  <arg name="child_frame_id" default="base_station"/>
  <arg name="UAV_NAME" default="$(optenv UAV_NAME uav1)"/>

  <group ns="$(arg UAV_NAME)">
    <node name="base_station_send_tf" pkg="mrs_ingeniarius_utils" type="base_station_send_tf">
        <param name="subscribed_topic" value="$(arg gnss_topic_in)"/>
        <param name="base_station_lat" value="$(arg base_station_latitude)"/>
        <param name="base_station_lon" value="$(arg base_station_longitude)"/>
        <param name="frame_id" value="$(arg frame_id)"/>
        <param name="child_frame_id" value="$(arg child_frame_id)"/>
    </node>
  </group> 

</launch>
